.github/workflows/review.yml

```diff
@@ -43,3 +43,26 @@ jobs:
       run: mv .github/tools/.checkpatch.conf .
     - name: Run checkpatch review
       uses: webispy/checkpatch-action@v9
+  upstream_tests:
+     runs-on: self-hosted
```

Comment:
take a look at the condition in the workflow of mdraid that make's sure the repo is exactly this one before using the self-hosted runner, we don't want forks using our runners.